specification/resources/resource-manager/Microsoft.Resources/stable/2020-06-01/resources.json

```diff
@@ -749,6 +749,44 @@
         "x-ms-odata": "#/definitions/DeploymentExtendedFilter"
       }
     },
+    "/providers/Microsoft.Management/managementGroups/{managementGroupId}/providers/{resourceProviderNamespace}/register": {
+      "post": {
+        "tags": [
+          "Providers"
+        ],
+        "operationId": "Providers_Register",
+        "description": "Registers a management group with a resource provider.",
+        "parameters": [
+          {
+            "name": "resourceProviderNamespace",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace of the resource provider to register."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/GroupIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the resource provider.",
+            "schema": {
+              "$ref": "#/definitions/Provider"
```

Comment:
Is this the right response schema for it? I see the API doesn't return a body on service side.